app/assets/stylesheets/application.css

```diff
@@ -74,14 +153,44 @@ body {
   margin-right: 5px;
 }
 
-.links {
-  color: white;
-}
-
 .margin {
   margin-left: 48px;
 }
 
 .links a {
   margin-left: 2px;
 }
+/* lkdnckdsnvknvkjnvksdjnvksdjnvkdsnvknsvkdsnvksnv */
+.post-comments {
```

Comment:
Separate out all the css related to `posts` into a separate file and require that here